src/vs-bicep/Bicep.VSLanguageServerClient.IntegrationTests/Utilities/WaitForExtensions.cs

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using Microsoft.Test.Apex.Services;
+
+namespace Bicep.VSLanguageServerClient.IntegrationTests.Utilities
+{
+    public static class WaitForExtensions
+    {
+        public static void IsTrue(Func<bool> predicate, TimeSpan timeout, Func<string>? onTimeout = null, string? conditionDescription = null)
+        {
+            bool result = WaitFor.TryIsTrue(predicate, timeout);
+
+            if (!result)
+            {
+                conditionDescription ??= "the condition to become true";
+
+                if (onTimeout != null)
+                {
+                    // Call the timout callback.
```

Comment:
```suggestion
                    // Call the timeout callback.
```

Reply:
Updated as part of this commit:
https://github.com/Azure/bicep/commit/415dad0122762c88b5683ef38cb70fd1625fecbd